src/templates/html/map/MapHTML.tsx

```diff
@@ -18,7 +19,14 @@ export function MapHTML(props: {
       <body>
         <Styles />
         <h1>Boundary</h1>
-        <Map boundary={props.geojson} clipGeojsonData={props.boundingBox} />
+        <Map
+          boundary={props.geojson}
+          clipGeojsonData={props.boundingBox}
+          userAction={props.userAction}
+        />
+        {props.userAction && (
+          <p>This PlanX user {props.userAction.toLowerCase()}.</p>
+        )}
```

Comment:
Conditional examples based on user action: 
![Screenshot from 2024-01-23 16-12-36](https://github.com/theopensystemslab/planx-core/assets/5132349/3861688e-7a86-4b80-b0b7-33105b6959bc)

(Second attribution line isn't displaying here like it should, need to debug this one - might be different style css?)
![Screenshot from 2024-01-23 16-51-44](https://github.com/theopensystemslab/planx-core/assets/5132349/405f9b62-d32f-421b-b6c7-1f358b26ae76)